complaintdatabase/tests.py

```diff
@@ -33,11 +38,12 @@ def test_get_context_data_exist(self, mock_flag_enabled):
         self.assertTrue('total_complaints' in response.context_data.keys())
         self.assertTrue('timely_responses' in response.context_data.keys())
 
+    @skipIf(not hasattr(settings, 'STANDALONE'), "not running standlone")
```

Comment:
This is true... although I wonder why we might ever set `STANDALONE=False`. It might be worth highlighting other potential side-effects with a failing test.  But I don't have strong feelings about that.